card/spec/lib/card/query_spec.rb

```diff
@@ -216,6 +218,7 @@
 
     it "should find right connection cards" do
       expect(Card::Query.new( :right=>"A" ).run.map(&:name).sort).to eq(["C+A", "D+A", "F+A"])
+      expect(Card::Query.new( :right=>{:content=>"Alpha [[Z]]"} ).run.map(&:name).sort).to eq(["C+A", "D+A", "F+A"])
```

Comment:
Line is too long. [116/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.<br>Space inside parentheses detected.